File: app/src/main/java/org/dhis2/usescases/programStageSelection/ProgramStageSelectionComponent.java
package org.dhis2.usescases.programStageSelection;

import org.dhis2.data.dagger.PerActivity;

import dagger.Subcomponent;

/**
 * Created by ppajuelo on 30/11/2017.
 *
 */
@PerActivity
@Subcomponent(modules = ProgramStageSelectionModule.class)
public interface ProgramStageSelectionComponent {
    void inject(ProgramStageSelectionActivity programStageSelectionActivity);
}

File: app/src/main/java/org/dhis2/usescases/programStageSelection/ProgramStageSelectionComponent.kt
package org.dhis2.usescases.programStageSelection

import dagger.Subcomponent
import org.dhis2.data.dagger.PerActivity

/**
 * Created by ppajuelo on 30/11/2017.
 *
 */
@PerActivity
@Subcomponent(modules = [ProgramStageSelectionModule::class])
interface ProgramStageSelectionComponent {
    fun inject(programStageSelectionActivity: ProgramStageSelectionActivity)
}
